### PR TITLE
pub docs.

### PR DESCRIPTION
**Pull request by dhatch**

### CODE REVIEW

**Review by samscott89 — CHANGES_REQUESTED** [review]
doesnt look right. before some mashes approve

**Review by samscott89 — APPROVED** [review]
I take it back `make ci-prod-docs` is good :)

File: .github/workflows/publish-docs.yml [hunk 1]
```diff
@@ -95,23 +93,14 @@ jobs:
...
         working-directory: docs
-
-      ### Push to S3 + invalidate cache
-      - name: s3 publish
-        run: aws s3 sync --delete docs/public/ s3://${{ github.event.inputs.url }}
-      - name: cloudfront invalidate docs preview
-        run: aws cloudfront create-invalidation --distribution-id E2KU2V8C9KJNU7 --paths "/*"
-        if: github.event.inputs.url == 'docs-preview.oso.dev'
-      - name: cloudfront invalidate docs
-        run: aws cloudfront create-invalidation --distribution-id E1MY35FKBOYAV1 --paths "/*"
```
Comment (samscott89) [inline .github/workflows/publish-docs.yml:116]: You'll want to keep this invalidation so that the root still invalidates

**Review by samscott89 — CHANGES_REQUESTED** [review]
Shall we delete all the docs-preview publishing steps? Those aren't really getting us anything since all branches are deployed to docs preview anyway.

**Comment by dhatch** [comment]
> Shall we delete all the docs-preview publishing steps? Those aren't really getting us anything since all branches are deployed to docs preview anyway.

Thought about it but I wanted to be able to test this job.

**Comment by dhatch** [comment]
Tested here: https://github.com/osohq/oso/runs/3680497127?check_suite_focus=true

**Review by samscott89 — APPROVED** [review]

File: .github/workflows/publish-docs.yml [hunk 2]
```diff
@@ -104,11 +104,15 @@ jobs:
...
+      - name: Install s3deploy
+        run: |
+          wget https://github.com/bep/s3deploy/releases/download/v2.4.0/s3deploy_2.4.0_Linux-64bit.deb
+          sudo dpkg -i s3deploy_2.4.0_Linux-64bit.deb
 
       ### Push to S3 + invalidate cache
-      - name: s3 publish
-        run: aws s3 sync --delete docs/public/ s3://${{ github.event.inputs.url }}
+      - name: deploy
+        run: s3deploy -bucket ${{ github.event.inputs.url }} -distribution-id E1MY35FKBOYAV1 -path v/main -source public -region us-east-1
```
Comment (samscott89) [inline .github/workflows/publish-docs.yml:114]:
```suggestion
        run: s3deploy -bucket ${{ github.event.inputs.url }}  -path v/main -source public -region us-east-1
```

Comment (samscott89) [reply .github/workflows/publish-docs.yml:114]: This would be better actually, a little less redundant since the next steps invalidate it again.

**Pull request merged**